content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -342,3 +513,5 @@ OLM performs dependency resolution at the namespace scope. It is possible to get
 `dependencies.yaml` is supported in `0.16.1+` versions of OLM.
 
 In versions of OLM < `0.16.1`, only GVK constraints are supported, and only via the `required` section of the ClusterServiceVersion.
+
+`properties.yaml` is supported in `0.17.4+` versions of OLM.
```

Comment:
The way this doc site works is that we have released versions of the site. What you're contributing to is the `master` branch. This is probably a good time to cut a 0.17.z release for olm too. So this comment is not a request to change anything, I think this looks good to be in the 0.17.z branch of the site. Leaving this here as a reminder to cut the 0.17.z release for the site.

Reply:
Yeah, that's a good call out Anik - I think my preference would be to always call once when net-new features are added to the documentation site, so I'd like to see us keep this note Vu added here. We don't do a great job of broadcasting our OLM release cadence, or what goes into an OLM minor release so being implicit with doc versioning might be challenging for a user to dictate when a feature was added, or whether they need to upgrade their OLM version to utilize the documentation functionality.